### ISSUE TITLE
Using sift.js with eval

### ISSUE THREAD

**Issue by marwej** [crcn/sift.js#154]
I have an issue with using sift.js in a sandbox using `eval` (which is using `vm`). Probably, I have missed to inject some needed object in the sandbox.

The second test fails (returning an empty array). Any suggestions why?

```javascript
import sift from 'sift';
import _eval from 'eval';

test('sift directly', () => {
  const sifter = sift({ a: 'a1' });
  const arr = [{ a: 'a1', b: 'b1' }, { a: 'a2', b: 'b2' }];
  const result = arr.filter(sifter);
  expect(result).toHaveLength(1);
});

test('sift with eval', () => {
  const code = `
    const sifter = sift({ a: 'a1' });
    const arr = [{ a: 'a1', b: 'b1' }, { a: 'a2', b: 'b2' }];
    module.exports = arr.filter(sifter);
  `;
  const result = _eval(code, 'filename', { sift });
  expect(result).toHaveLength(1);
});
```

**Comment by crcn** [comment]
Added a fix in `8.3.4`. Let me know if that works 🙂.

**Comment by marwej** [comment]
Wonderful, thank you!!
